d2l-input-textarea.html

```diff
@@ -45,6 +45,23 @@
 				@apply --d2l-input-textarea;
 			}
 
+			/* fix IE+ textarea content cut off b/c mirror text size is smaller than textarea which mirror text is borderless */
+			@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+     			/* IE10+ CSS styles go here */
+				.mirror-text {
+					border-style: solid !important;
```

Comment:
I'm not really following what this PR is fixing. Can you clarify and maybe show a quad site link or something where it is failing with an example of the content that causes it to fail. Is the problem occurring for all auto grow textareas or only the ones we use in rubrics where we hide the regular border but then show a border on hover/focus.

The reason I'm confused is because for the IE/Edge fixes, for a regular textarea the border width should already be set to 1px by the default d2l-input styles.

If this is just for the no-border scenario, then I'd like to understand a bit better why it is not working. In my experience, problems arise with the auto grow text area when the styles of the mirror and the underlying textarea don't match exactly. So the strategy has been to set the border to be the same regardless of focus styles, and then adjust the padding on focus just on the textarea. The padding on the mirror shouldn't need to change as it is the combined border + padding that determines the amount of text that will fit correctly.